tensorflow_addons/image/sparse_image_warp_test.py

```diff
@@ -26,6 +26,46 @@
 from tensorflow_addons.utils.resource_loader import get_path_to_datafile
 
 
+def testZeroShift():
```

Comment:
The pep8 convention recommends snake_case for functions and methods names. In unittest, they don't respect this convention, but since we're going for pytest now, it's a good opportunity for starting to be pep8 compliant.

Reply:
Thanks for the reminder @gabrieldemarmiesse.